github/git_refs.go

```diff
@@ -45,7 +46,8 @@ type updateRefRequest struct {
 	Force *bool   `json:"force"`
 }
 
-// GetRef fetches the Reference object for a given Git ref.
+// GetRef fetches a single Reference object for a given Git ref.
+// If a Git ref does not match exactly, this method will return a nil result.
```

Comment:
I would rephrase this slightly:

"If a Git ref does not match exactly" -> "If there is no exact match"
"this method" -> "GetRef"
"nil result" -> "nil Reference"

So:

```
// If there is no exact match, GetRef will return a nil Reference.
```